backend/src/lib/config/env.ts

```diff
@@ -204,6 +204,18 @@ const envSchema = z
     WORKFLOW_SLACK_CLIENT_SECRET: zpStr(z.string().optional()),
     ENABLE_MSSQL_SECRET_ROTATION_ENCRYPT: zodStrBool.default("true"),
 
+    // Special Detection Feature
+    PARAMS_FOLDER_SECRET_DETECTION_PATHS: zpStr(
+      z
+        .string()
+        .optional()
+        .transform((val) => {
+          if (!val) return undefined;
+          return JSON.parse(val) as { secretPath: string }[];
+        })
+    ),
+    PARAMS_FOLDER_SECRET_DETECTION_ENABLED: zodStrBool.default("false"),
```

Comment:
if they have PARAMS_FOLDER_SECRET_DETECTION_PATHS do we need PARAMS_FOLDER_SECRET_DETECTION_ENABLED still?